ITEM.tsv

```diff
@@ -5567,11 +5567,11 @@ ITEM_20150717_005566	The first time a device was attached to the tip of a bow to
 ITEM_20150717_005567	This bow requires a lot more strength from the user than expected. The view of the arrow in flight will leave a sense of satisfaction.
 ITEM_20150717_005568	Unlike its name, it will be welcomed by any archer. Only rogues dislike it.
 ITEM_20150717_005569	The burden of bows has been alleviated, it does not decrease the power by much.
-ITEM_20150717_005570	- All Hunter Skill Levels increased by +2
+ITEM_20150717_005570	- Increased all Hunter skill levels by 2
 ITEM_20150717_005571	- Basic Attacks have 2% chance to poison the target for 20 seconds
-ITEM_20150717_005572	If you don't have the other bow, then this bow is good choice.
-ITEM_20150717_005573	Weapon Merchant said that even Lydia Scaffen seems to be buying it. It is not wrong saying since she is never complain the bow.
-ITEM_20150717_005574	- 100% Critical chance to Beast types{nl} - 25% decreased damage to non-Beast types
+ITEM_20150717_005572	If you don't have the other bow, then this bow is good choice to buy.
```

Comment:
"then this bow is good choice to buy." -> "then this bow is a good choice to buy."